docs/ruleset_functions/get_zone_target_baseline_system.md

```diff
@@ -0,0 +1,97 @@
+# get_zone_target_baseline_system
+**Schema Version:** 0.0.22  
+
+**Description:** Following G3.1.1, determines the baseline system type for each zone in a building
+
+**Inputs:**
+- **P-RMD**
+- **B-RMD**
+
+**Returns:**  
+- **zones_and_systems**: a dictionary with zone / list pairs where the first value in the list is the expected system type (ex "SYS-3") and the second value is the rule used to choose the system, (eg "G3_1_1e"): zones_and_systems[zone] = ["SYS-3", "G3_1_1e"]
+ 
+**Function Call:**
+- **get_HVAC_building_area_types_and_zones**
+- **does_zone_meet_G3_1_1b**
+- **does_zone_meet_G3_1_1c**
+- **does_zone_meet_G3_1_1e**
+- **does_zone_meet_G3_1_1f**
+- **does_zone_meet_G3_1_1g**
+- **get_predominant_HVAC_building_area_type**
+- **get_number_of_floors**
+- **expected_system_type_from_Table_G3_1_1**
+
+**Data Lookup**
+- **Table G3.1.1**
+
+## Logic:
+- get climate zone (not sure this is the correct path): `climate_zone = ASHRAE229.weather.climate_zone`
+- create zones_and_systems list: `zones_and_systems = {}`
+- pre-fill the list with an empty string:
+	- `for building_segment in RMR.building.building_segments:`
+		- `for zone in building_segment.zones:`
+			- `zones_and_systems[zone] = []`
+- get the list of HVAC building area types and zones: `list_building_area_types_and_zones = get_HVAC_building_area_types_and_zones(B-RMR)`
+- get the predominant building area type: `predominant_building_area_type = get_predominant_HVAC_building_area_type(list_building_area_types_and_zones)`
+- get the total number of floors for the building: `num_floors = get_number_of_floors(B-RMR)`
+- first get the expected system type for the predominant building area type.  To do this, we need the total area of the predominant building area type, which includes all BAT's less than 20000 ft2
+- set area equal to the area of the predominant building area type: `area = list_building_area_types_and_zones[predominant_building_area_type]["AREA"]`
+- loop through the building area types: `for bat in list_building_area_types_and_zones:`
+	- if the bat is not equal to the predominant building area type: `if bat != predominant_building_area_type:`
+		- and if the area of this bat < 20,000 ft2: `if list_building_area_types_and_zones[bat]["AREA"] <= 20000:`
```

Comment:
logic is not consistent in the description vs. in the pseudo code. Should this be `< 20000` ?

Reply:
The logic is correct.  Description is wrong.